### PR TITLE
Implement support for GSSAPI extension gss_inquire_sec_context_by_oid

### PR DESCRIPTION
**Pull request by vm86**
Hello, I added a raw function : gss_inquire_sec_context_by_oid .

From this issues:
https://github.com/pythongssapi/python-gssapi/issues/51

was tested on the SMB protocol.

### CODE REVIEW

File: gssapi/raw/ext_ggf.pyx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    output_token_buffer_ptr = &output_token_buffer
+
+    cdef gss_OID_set desired_mechs
+
+    with nogil:
+        maj_stat = gss_inquire_sec_context_by_oid(&min_stat, context.raw_ctx, &mech.raw_oid, output_token_buffer_ptr)
+
+    if maj_stat == GSS_S_COMPLETE:
+        py_token = []
```
Comment (DirectXMan12) [inline gssapi/raw/ext_ggf.pyx]: nit: `py_tokens` for clarity.

File: gssapi/raw/ext_ggf.pyx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+cdef extern from "python_gssapi_ext.h":
+    OM_uint32 gss_inquire_sec_context_by_oid(OM_uint32 *min_stat,
+                                  const gss_ctx_id_t context_handle,
+                                  const gss_OID desired_object,
+                                  gss_buffer_set_t *data_set) nogil
+
+
+def inquire_sec_context_by_oid(SecurityContext context not None, OID mech not None):
+    """
+    Args:
```
Comment (DirectXMan12) [inline gssapi/raw/ext_ggf.pyx]: This needs to follow the proper documentation format, including listing what it does, and what it returns.  In general, `python-gssapi` documentation should be more-or-less usable without having to reference the main RFC.

**Comment by frozencemetery** [comment]
Hi @vm86, are you still working on this?  Is there anything we can help with?

**Comment by vm86** [comment]
Hi @frozencemetery, yes, I work with, but heaven't time to do it. I need to write or find documentation for gss_inquire_sec_context_by_oid. Can you help me with?

**Comment by frozencemetery** [comment]
Unfortunately, this function doesn't seem to have been documented on it's addition.  When that happens, we tend to read the source or look for code that uses it.  Then when other people want to know what a GSSAPI function does, they can come look at our documentation :)

Since presumably you need this function for something, you probably know what it does, right?  Take a look at the docs for similar function [inquire_cred_by_mech](https://github.com/pythongssapi/python-gssapi/blob/master/gssapi/raw/creds.pyx#L354-L381) and try to match that.

**Comment by vm86** [comment]
I close, as done in: https://github.com/pythongssapi/python-gssapi/pull/141

**Pull request closed without merging**